### DISCUSSION TITLE
How to get field value(state) in component props

### DISCUSSION THREAD

**Discussion by nhattran998** [alibaba/formily#3240] [Q&A 2.x] [answered]
I currently integrate @formily/react 2.1.9 with MUI(Material UI). But I have a problem with the error and helper text of MUI.
When I submit the form, because the form is not reactive, I cannot get the form error state to display to UI. 

Normally, if I use the native `input`, I can get this stuffs with `decorator` + `useField()`. 
So hopefully the maintainer can help me 🙏. Thanks!

```
<Field
      name="email"
      component={[
        MuiInput,
        {
          id: 'input-register-email',
          type: 'email',
          name: 'email',
          sx: { height: 50, py: 4, px: 3, borderRadius: '48px' },
          error: form.errors.length > 0 || ||  form.getFormState().errors.length > 0,
          helperText: form.errors[0].messages ||  form.getFormState().errors[0].messages
                }
              ]}
            />
       ```

**Comment by janryWang** [comment, marked as answer]
You have to make a FormItem component to display the error message
